accepted/0000-yomi-initial-integration.md

```diff
@@ -0,0 +1,427 @@
+- Feature Name: Initial integration with Yomi (MVP)
+- Start Date: 2019-10-08
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Yomi (Yet one more installer) is an installer for openSUSE (Leap,
+Tumbleweed, MicroOS) and SLE (WIP) expressed as a set of Salt
+states. This RFC propose the integration of Yomi for provisioning of
+new systems in Uyuni.
+
+# Motivation
+[motivation]: #motivation
+
+Currently in Uyuni / SUMA we have several ways for installing a new
+system:
+
+- For RedHat based systems we can use the Web UI to create, update and
+  manage Kickstart files. Those files can be designed to be applied
+  for different nodes. (IS THIS TRUE?)
+
+- For [open]SUSE based systems we can upload an AutoYaST file designed
+  for a single type on system.
+
+- We can leverage Cobbler manage the PXE-Boot server and automate the
+  installation of new systems (via Kickstart, AutoYaST and Cheetah or
+  Jinja2 template engine)
+
+- SUMA Manager for Retail allows the installation of new systems
+  dumping a partition image (generated by KIWI from a given template)
+  into the local root partition. For this case the PXE boot stack will
+  be different that the one provided by Cobbler
```

Comment:
I don't know why the future tense. PXE boot stack is different for saltboot. The functionality was already shipped with SUMA 3.2 last year.
Maybe add a note that for it PXE is managed by other formulas provided with SUMA Retail.

Reply:
Fixed.